test/ejabberd_tests/tests/amp_SUITE.erl

```diff
@@ -804,16 +815,16 @@ amp_el([]) ->
     throw("cannot build <amp> with no rules!");
 amp_el(Rules) ->
     #xmlel{name = <<"amp">>
-          ,attrs = [{<<"xmlns">>, ns_amp()}]
-          ,children = [ rule_el(R) || R <- Rules ]
+          , attrs = [{<<"xmlns">>, ns_amp()}]
```

Comment:
Since you touched line like this, could you please move the comma to the end of prev line? In this project we rather avoid commas at the beginning of a line.